spec/models/proposal_spec.rb

```diff
@@ -112,6 +112,15 @@
       proposal.add_observer(observer)
       expect(proposal.users).to eq [observer]
     end
+
+    it "can exclude delegates" do
+      delegate_one = create(:user)
+      delegate_two = create(:user)
+      proposal = create(:proposal, :with_approver)
+      proposal.approvers.first.add_delegate(delegate_one)
+      proposal.approvers.first.add_delegate(delegate_two)
+      expect(proposal.users).to match_array(proposal.users_except_delegates + [delegate_one, delegate_two])
```

Comment:
not trying to be a nit picker here, but if I were looking for a test for this method, I would expect it to be within a `describe "#method_name` block as opposed to embedded in a test for another method